topics/metagenomics/tutorials/mothur-miseq-sop/tours/tour.yaml

```diff
@@ -0,0 +1,1984 @@
+      within Mothur, using the <b>chimera.uchime</b> command. This command will
+      split the data by sample and check for chimeras.    backdrop: true
+
+      <br>Our preferred way of doing this is to use the abundant sequences as
+      our reference. In addition, if a sequence is flagged as chimeric in one
+      sample, the default (dereplicate=No) is to remove it from all samples. Our
+      experience suggests that this is a bit aggressive since we’ve seen rare
+      sequences get flagged as chimeric when they’re the most abundant sequence
+      in another sample
+    backdrop: true
+  - title: Step 46. Remove chimeric sequences
+    element: '#tool-search-query'
+    content: Search for Chimera.uchime tool
+    placement: right
+    textinsert: Chimera.uchime
+  - title: Step 47. Remove chimeric sequences
+    element: '#tool-search'
+    content: Click on the "Chimera.uchime" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_uchime%2Fmothur_chimera_uchime%2F1.36.1.0"]
+  - title: Step 48. Remove chimeric sequences
+    element: '#tool-search'
+    content: |-
+      Execute the tool with  <ul>
+        <li>“fasta” to the fasta output from Pre.cluster</li>
+        <li>“Select Reference Template from” to <b>Self</b></li>
+        <li>“count” to the count table from the last Pre.cluster</li>
+        <li>“dereplicate” to Yes</li>
+      </ul>
+    position: left
+  - title: Step 49. Remove chimeric sequences
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      Running chimera.uchime with the count file will remove the chimeric
+      sequences from the count table, but we still need to remove those
+      sequences from the fasta file as well    position: left
+    position: left
+  - title: Step 50. Remove chimeric sequences
+    element: '#tool-search-query'
+    content: Search for Remove.seqs tool
+    placement: right
+    textinsert: Remove.seqs
+  - title: Step 51. Remove chimeric sequences
+    element: '#tool-search'
+    content: Click on the "Remove.seqs" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_seqs%2Fmothur_remove_seqs%2F1.36.1.0"]
+  - title: Step 52. Remove chimeric sequences
+    element: '#tool-search'
+    content: |-
+      Execute the tool with  <ul>
+        <li>“accnos” to the uchime.accnos file from Chimera.uchime</li>
+        <li>“fasta” to the fasta output from Pre.cluster</li>
+        <li>“count” to the count table from Chimera.uchime</li>
+      </ul>
+    position: left
+  - title: Question. Remove chimeric sequences
+    content: |-
+      Inspect the output <ul>
+        <li>How many sequences were flagged as chimeric? what is the percentage? (Hint: summary.seqs)</li>
+      </ul>
+    backdrop: true
+  - title: Step 51. Remove chimeric sequences
```

Comment:
Do we really need to put step numbers?

Reply:
as far as I'm concerned no, it seems needlessly tedious to shift all number if we ever decide to add/remove a step (btw it's not just the numbering that's off, the steps themselves are repeated)

Reply:
I was thinking that number is helpful to navigate, since, for example, uploading data process consists of ~6 steps. But I can remove them

Reply:
Ping @bebatut. Maybe it would make sense for some tasks, using "Step 1/N".

Reply:
If so, this should be part of the framework and not part of the documentation imho. So removing it from here make sense to me.

Reply:
I would be to remove the step numbering. It would be difficult to keep up-to-date in the future. 
@polkhe can you remove them? Same for the other tour in this PR? Thanks a lot